docs/source/using/managing-minishift.adoc

```diff
@@ -53,6 +53,32 @@ the GitHub issue https://github.com/minishift/minishift/issues/179[#179].
 The link:../command-ref/minishift_delete{outfilesuffix}[`minishift delete`] command deletes the OpenShift cluster,
 and also shuts down and deletes the Minishift VM. No data or state are preserved.
 
+[[choosing-vm]]
+== Choosing the VM image
```

Comment:
I considered that, but I figured if we want to really sign-post it, then it should have a 2heading and therefore show up in the TOC of the page. It's also something that you can persist IIRC so that means it's relevant to other sections in the page.